localisation/00_wc_vanilla_replacements.csv

```diff
@@ -255,4 +255,9 @@ bloodthirsty_mass_sacrifice_bonus_3_desc;By appeasing [Root.Religion.GetHighGodN
 EVTDESCAHF23463;[From.GetTitledFirstName] has performed a great sacrifice, slaughtering hundreds of our own people to garner the favor of [From.Religion.GetHighGodName] during this war. With such a large offering, we are sure to prevail against our foes.;;;;;;;;;;;;;x
 EVTDESCAHF23464;[From.GetTitledFirstName] has performed a great sacrifice, slaughtering hundreds of our own people to garner the favor of [From.Religion.GetHighGodName] during this war. [From.GetHerHisCap] fanatic obsession with sacrifices will doom us all.;;;;;;;;;;;;;x
 EVTDESCAHF23465;[From.GetTitledFirstName] has performed a great sacrifice, slaughtering hundreds of [Root.GetHerHis] own people to garner the favor of [From.Religion.GetHighGodName] during this war. [From.GetSheHeCap] hopes that the Blood God will now grant [From.GetHerHis] armies the strength necessary to defeat my warriors.;;;;;;;;;;;;;x
-EVTOPTAHF23465;What if [From.Religion.GetHighGodName] listened to [From.GetHerHim]?;;;;;;;;;;;;;x
\ No newline at end of file
+EVTOPTAHF23465;What if [From.Religion.GetHighGodName] listened to [From.GetHerHim]?;;;;;;;;;;;;;x
+hermetics_write_magnum_opus;Write Down Whispers;;;;;;;;;;;;;x
+hermetics_write_magnum_opus_desc;Write down whispers of Old Gods that constantly ring in your ears.;;;;;;;;;;;;;x
+hermetics_write_magnum_opus_power_desc;Enables �Gright-clicking�! on your own portrait in order to write down in a book mysterious whispers of your masters.;;;;;;;;;;;;;x
```

Comment:
Enables �Gright-clicking�! on your own portrait in order to write down the mysterious whispers of your masters in a book.